projects/packages/my-jetpack/_inc/data/evaluation-recommendations/use-evaluation-recommendations.ts

```diff
@@ -30,6 +25,23 @@ const useEvaluationRecommendations = () => {
 		'recommendedModules',
 		getInitialRecommendedModules()
 	);
+
+	const unownedRecommendedModules = useMemo( () => {
+		const { ownedProducts = [] } = getMyJetpackWindowInitialState( 'lifecycleStats' );
+		// We filter out owned modules, and return top 3 recommendations
+		return recommendedModules?.filter( module => ! ownedProducts.includes( module ) ).slice( 0, 3 );
+	}, [ recommendedModules ] );
+
+	const isEligibleForRecommendations = useMemo( () => {
+		const { dismissed } = getMyJetpackWindowInitialState( 'recommendedModules' );
+		return ! dismissed && ! isWelcomeBannerVisible && isJetpackUserNew();
+	}, [ isWelcomeBannerVisible ] );
+
+	const [ isSectionVisible, setIsSectionVisible ] = useValueStore(
+		'recommendedModulesVisible',
+		isEligibleForRecommendations && !! unownedRecommendedModules?.length
```

Comment:
Is the `!!` necessary here?

Reply:
Imo yes, otherwise we'll return `0` 🤔

Reply:
Oh no you're right, I read that wrong. No need to change it, thank you!